### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by kyrretl**
PR for the rest of the problems we ran into in our October workshop at the University of Oslo

Please see the commit messages for details

### CODE REVIEW

File: _episodes/05-null.md [hunk 1]
```diff
@@ -226,6 +226,16 @@ detail in [the next section]({{ site.github.url }}/06-agg/).
 >
 > to produce?
 > What does it actually produce?
+> > ## Solution
+> >
+> > The IN () function does not match NULL, and we only get results for the date '1927-02-08'.
```
Comment (danmichaelo) [inline _episodes/05-null.md]: Perhaps it's even good to recap what IN really does at this point:
```suggestion
> > We only get a result for the value '1927-02-08', not for NULL. This is because `dated IN ('1927-02-08', NULL)` is just a shorthand for `dated = '1927-02-08' OR dated = NULL`, and we now know that the latter condition (`dated = NULL`) is never true. 
```

Comment (danmichaelo) [reply _episodes/05-null.md]: We can get back to this in a separate issue

**Comment by kyrretl** [comment]
@henrykironde, do you have a chance to look at this? The changes should be quite straight-forward. We're teaching this workshop again soon, so would be great to have the changes merged.

File: _episodes/06-agg.md [hunk 2]
```diff
@@ -355,10 +355,7 @@ this query:
 > >
 > > You can confirm this, by executing this code:
 > > ```
-> > SELECT AVG(a) FROM (
-> >     SELECT 1 AS a
-> >     UNION ALL SELECT NULL
-> >     UNION ALL SELECT 5);
+> > SELECT AVG(a) FROM (SELECT 1 AS a UNION SELECT NULL UNION SELECT 5);
```
[the remark this answers is not in the document]
Comment (henrykironde) [reply _episodes/06-agg.md]: We could use  ~~~ vs ``` to keep the consistency

File: _episodes/06-agg.md [hunk 3]
```diff
@@ -376,22 +373,37 @@ this query:
...
+> > {: .sql}
+> >
+> > This produces the average value (6.5625), which we can then insert into a second query:
+> >
+> > ~~~
+> > SELECT reading - 6.5625 FROM Survey WHERE quant = 'rad';
+> > ~~~
+> > {: .sql}
+> >
+> > This produces what we want! But, lo and behold, we can combine this into a single query using something called subqueries!
```
Comment (henrykironde) [inline _episodes/06-agg.md]: I think this sounds better 
`This produces what we want, but we can combine this into a single query using subqueries.`

File: _episodes/06-agg.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kyrretl) [reply _episodes/06-agg.md]: Thanks for noticing! Upon fixing this, we found that it was perhaps better to remove UNION altogether here, since it's only briefly introduced earlier (as a challenge in episode 4: https://github.com/swcarpentry/sql-novice-survey/search?q=union&unscoped_q=union), and is potentially confusing here.

We're a bit unsure if UNION need be introduced at all in this introductory course, but that's perhaps better to discuss in a separate issue. Let us know if you have thoughts on this though.

File: _episodes/06-agg.md (hunk 3 above)
Comment (kyrretl) [reply _episodes/06-agg.md]: Agree

**Comment by kyrretl** [comment]
Thanks for reviewing, @henrykironde ! We've made changes to the areas requested, see comments above for details.

File: _episodes/10-prog.md [hunk 4]
```diff
@@ -389,3 +389,23 @@ Full name for barrett: Mary Barrett
 > >
 > {: .solution}
 {: .challenge}
+
```
Comment (henrykironde) [inline _episodes/10-prog.md]: It would be really great if we can put this in a separate PR.

File: _episodes/09-create.md [hunk 5]
```diff
@@ -104,10 +104,13 @@ INSERT INTO Site (name, lat, long) VALUES ('MSK-4', -48.87, -123.40);
 ~~~
 {: .sql}
 
+Note that you have to insert a correct value, defined by the datatype, into every column - you can't refrain from inserting into a column, 
+if you have no value to insert use NULL.
```
Comment (henrykironde) [inline _episodes/09-create.md]: It would be really great if we can put this whole file in a separate PR.

File: _episodes/05-null.md [hunk 6]
```diff
@@ -235,6 +235,16 @@ detail in [the next section]({{ site.github.url }}/06-agg/).
 >
 > to produce?
```
[the remark this answers is not in the document]
Comment (henrykironde) [reply _episodes/05-null.md]:
```
> ~~~
> {: .sql}
>
> to produce?
> 
> What does it actually produce?
```

**Comment by danmichaelo** [comment]
It's now split into #315, #316, #317, #318 + #304 , so this PR can now be closed.
